unittests/TestModuleFactory.cpp

```diff
@@ -66,68 +66,48 @@ static std::unique_ptr<Module> parseIR(const char *IR) {
 
 #pragma mark - Negate Condition
 
-std::unique_ptr<Module> TestModuleFactory::create_SimpleTest_NegateCondition_Tester_Module() {
-  std::string contents = createFixture("fixture_simple_test_negate_condition_operator_tester.ll");
-
+std::unique_ptr<Module> TestModuleFactory::createModule(const char *fixtureName, const char *moduleIdentifier) {
+  std::string contents = createFixture(fixtureName);
+  
   auto module = parseIR(contents.c_str());
-
-  module->setModuleIdentifier("fixture_simple_test_negate_condition_operator_tester.ll");
-
+  
+  module->setModuleIdentifier(moduleIdentifier);
+  
   return module;
 }
 
-std::unique_ptr<Module> TestModuleFactory::create_SimpleTest_NegateCondition_Testee_Module() {
-  std::string contents = createFixture("fixture_simple_test_negate_condition_operator_testee.ll");
-
-  auto module = parseIR(contents.c_str());
-
-  module->setModuleIdentifier("fixture_simple_test_negate_condition_operator_testee.ll");
+std::unique_ptr<Module> TestModuleFactory::create_SimpleTest_NegateCondition_Tester_Module() {
+  const char *fixture = "fixture_simple_test_negate_condition_operator_tester.ll";
+  return createModule(fixture, fixture);
+}
 
-  return module;
+std::unique_ptr<Module> TestModuleFactory::create_SimpleTest_NegateCondition_Testee_Module() {
+  const char *fixture = "fixture_simple_test_negate_condition_operator_testee.ll";
+  return createModule(fixture, fixture);
 }
 
 #pragma mark - Remove Void Function
 
 std::unique_ptr<Module> TestModuleFactory::create_SimpleTest_RemoveVoidFunction_Tester_Module() {
-  std::string contents = createFixture("fixture_simple_test_remove_void_function_operator_tester.ll");
-
-  auto module = parseIR(contents.c_str());
-
-  module->setModuleIdentifier("fixture_simple_test_remove_void_function_operator_tester.ll");
-
-  return module;
+  const char *fixture = "fixture_simple_test_remove_void_function_operator_tester.ll";
+  return createModule(fixture, fixture);
 }
 
 std::unique_ptr<Module> TestModuleFactory::create_SimpleTest_RemoveVoidFunction_Testee_Module() {
-  std::string contents = createFixture("fixture_simple_test_remove_void_function_operator_testee.ll");
-
-  auto module = parseIR(contents.c_str());
-
-  module->setModuleIdentifier("fixture_simple_test_negate_condition_operator_testee.ll");
-
-  return module;
+  return createModule("fixture_simple_test_remove_void_function_operator_testee.ll",
+                      "fixture_simple_test_negate_condition_operator_testee.ll");
```

Comment:
fixture_simple_test_remove_void_function_operator_testee.ll 

vs 

fixture_simple_test_negate_condition_operator_testee.ll (should be allso remove_void_function...)

follow the same pattern (fixture, fixture) as you followed above and below.